Preferences.tsx

```diff
@@ -13,16 +13,14 @@ import {z} from 'zod';
 export const PREFERENCES_KEY = 'PREFERENCES';
 
 const preferencesSchema = z.object({
-  center: avalancheCenterIDSchema,
-  hasSeenCenterPicker: z.boolean(),
+  center: avalancheCenterIDSchema.default('NWAC'),
+  hasSeenCenterPicker: z.boolean().default(false),
+  secretMenuCollapsed: z.boolean().default(true),
```

Comment:
before, you had to add a field here and then remember to add it to `defaultPreferences`. now you just add a field once (and if you forget to set a default, line 23 will throw an exception immediately)